bHY6yh-d38247/cod-bxxqqq_bHY6yh-d38247.xml

```diff
@@ -0,0 +1,176 @@
+              pro amico tuo defix<lb ed="#B" break="no"/>i  sti 
+              apud extraneum animam tuam illa<lb ed="#B" break="no"/>q  ueatus 
+              es verbis oris tui Et 
+              <lb ed="#B"/>captus propriis sermonibus tuis / 
+              <lb ed="#B"/> ¶ fac ergo quod dico temet ipsum libe<lb ed="#B" break="no"/>r  a 
+              quia incidisti in manus proximi 
+              <lb ed="#B"/>tui discurre festina suscita ami<lb ed="#B" break="no"/>c  um 
+              tuum nec dederis sompnum 
+              <lb ed="#B"/>oculis tuis nec dormient pal<lb ed="#B" break="no"/>p  ebre 
+              tue Erue quasi damula 
+              <lb ed="#B"/>de manu et quasi auis de insidiis 
+              <lb ed="#B"/>aucupis</quote>
+            </p>
+            <p xml:id="bHY6yh-d38247-d1e159">
+               ¶ <ref xml:id="bHY6yh-d38247-Rd1e157">22 capitulo</ref> <quote xml:id="bHY6yh-d38247-Qd1e161">Qui accipit mu<lb ed="#B" break="no"/>t  uum
+              seruus est fenerantis</quote>
+            </p>
+            <p xml:id="bHY6yh-d38247-d1e167">
+              <lb ed="#B"/><ref xml:id="bHY6yh-d38247-Rd1e169">IEsus filius syrach in ecclesiastico capitulo 29</ref> <quote xml:id="bHY6yh-d38247-Qd1e171">fene<lb ed="#B" break="no"/>r  are
+              tibi proximo in tempore neces<lb ed="#B" break="no"/>s  itatis 
+              illius  ¶ Redde proximo in tempore suo 
+              <lb ed="#B"/> ¶ Multi non causa nequitie non fene<lb ed="#B" break="no"/>r  ati 
+              sunt sed fraudari gratis timu<lb ed="#B" break="no"/>e  runt 
+               ¶ vir bonus fidem facit proximo suo</quote>
+            </p>
+            <p xml:id="bHY6yh-d38247-d1e183">
+              <lb ed="#B"/><ref xml:id="bHY6yh-d38247-Rd1e187">MAtheus in ewangelio capitulo 22</ref> <quote xml:id="bHY6yh-d38247-Qd1e187">Reddite 
+              <lb ed="#B"/>que sunt cesaris cesari et que 
+              <lb ed="#B"/>sunt dei deo</quote>
+            </p>
+            <p xml:id="bHY6yh-d38247-d1e193">
+               ¶ <ref xml:id="bHY6yh-d38247-Rd1e197">Idem dicit marcus 
+              <lb ed="#B"/>in ewangelio capitulo 11</ref> <quote xml:id="bHY6yh-d38247-Qd1e195"></quote>
+            </p>
+            <p xml:id="bHY6yh-d38247-d1e201">
+               ¶ <ref xml:id="bHY6yh-d38247-Rd1e207">Idem dicit lucas in ewangelio 
+              <lb ed="#B"/>capitulo 20</ref> <quote xml:id="bHY6yh-d38247-Qd1e203"> </quote>
+            </p>
+            <p xml:id="bHY6yh-d38247-d1e215">
+              <lb ed="#B"/><ref xml:id="bHY6yh-d38247-Rd1e226">PAulus<!--line jumps up--> apostolus epistola ad romanos<!--line jumps down--> capitulo 13</ref>  ¶ <quote xml:id="bHY6yh-d38247-Qd1e219">Reddite omnibus 
+              <lb ed="#B"/>debita cui tribulum tributum 
+              <lb ed="#B"/>cui vectigal vectigal. cui hono<lb ed="#B" break="no"/>r  em 
```

Comment:
there's still a problem with the linebreaks here and throughout

Reply:
fixed it